### ISSUE TITLE
UI Customization

### ISSUE THREAD

**Issue by remboshelby** [Tangem/tangem-sdk-flutter#2]
For full integration into the project, I need to customize the UI interface responsible for reading data from the Tangem card.

In the native versions of the SDK (Android / iOS), the readme contains an item with information on customizing the SDK. Please share some information on Flutter.

**Comment by lazutkin-andrey** [comment]
Hey @remboshelby,
We're sorry, but our Flutter plugin does not support UI customization for now.
Anyway, let us know what you want to achieve and we'll try to come up with a solution in the nearest future.

**Comment by remboshelby** [comment]
In our project, it is necessary to customize the widget that is responsible for the user's request to lean the Tangem card against the phone with its icon and animation.

**Comment by lazutkin-andrey** [comment]
Yeah, I've got it. I mean what kind of customizations do you need? Just to add a "night theme", or ability to change the background and other elements colors, or to be able change the behavior, animation and text?

**Comment by remboshelby** [comment]
We need to implement a dark / light theme, change the title text, change image and animation.
